supervisor/host/network.py

```diff
@@ -82,15 +82,16 @@ def dns_servers(self) -> list[str]:
 
         return list(dict.fromkeys(servers))
 
-    async def check_connectivity(self):
+    async def check_connectivity(self, *, force: bool = False):
         """Check the internet connection."""
         if not self.sys_dbus.network.connectivity_enabled:
+            self.connectivity = None
```

Comment:
Is this a bad idea? The value of this field can be `None` but normally only at startup until a connectivity check either succeeds or fails. I wanted to revert it back to `None` if someone disabled the connectivity check but doing so will cause supervisor to fire an event from here withe value set to `None`:
https://github.com/home-assistant/supervisor/blob/dcd0592d44026c0e0aa6223ac306e0ccf9f9c20d/supervisor/host/network.py#L60-L62

I don't think we did that before, I think every time this event fired it was value True/False since it never changed back to None. I don't know who listens to this event so I can't check if that possibility is handled.

Reply:
I think with this PR, we can support that we not support to disable that. We supported disable it because of old NM version from debian 9 or 10

Reply:
Ah ok so you're thinking add a check for this property. Yea I can do that

Reply:
Ok I added an evaluation for disabling the NM connectivity check @pvizeli